BeyondCorpAppConnections/.repo-metadata.json

```
@@ -0,0 +1,6 @@
+{
+    "distribution_name": "google/cloud-beyondcorp-appconnections",
+    "release_level": "beta",
+    "client_documentation": "http://googleapis.github.io/google-cloud-php/#/docs/cloud-beyondcorp-appconnections/latest",
+    "library_type": "GAPIC_AUTO"
```

FYI: We might want to add more repo metadata like [python does](https://github.com/googleapis/python-storage/blob/main/.repo-metadata.json). I am not sure where all this gets consumed, but for #5452, we are going to start looking for `issue_tracker` and `product_documentation`.
```suggestion
    "library_type": "GAPIC_AUTO",
    "product_documentation": "https://cloud.google.com/beyondcorp-enterprise/docs",
    "issue_tracker": "https://github.com/googleapis/google-cloud-php/issues"
```